src/main/java/seedu/address/storage/JsonAdaptedStepDescription.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.step.Step;
+
+/**
+ * Jackson-friendly version of {@link Step}.
+ */
+class JsonAdaptedStepDescription {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Step's %s field is missing!";
+
+    private final String stepDescription;
+
+    /**
+     * Constructs a {@code JsonAdaptedStep} with the given {@code stepDescription}.
+     */
+    @JsonCreator
+    public JsonAdaptedStepDescription(@JsonProperty("description") String stepDescription) {
```

Comment:
Sorry I think I wasn't clear previously, I meant that there's this `@JsonProperty` which seems unnecessary, because each step will be stored like `{"description": "step"}` and the list of steps in recipe would be like `[{"description": "step 1"}, {"description": "step 2"}, ...]`. So I was thinking maybe we could just do without a property name so that the list of steps would be more like `["step 1", "step 2", ...]`. Then the code would look something like the current `JsonAdaptedTag`.

Reply:
Alright I see. Side note, should the step number be stored in the Json as well, or do we solely rely on UniqueStepList to do that?

Reply:
I think we can keep it like this for now, otherwise we would need implement and test the checking of order of steps, and deal with the exceptions from that. Maybe we can consider if we have time after v1.3.

Reply:
It seems like a `@JsonValue` is now required for serialization into a Json file, otherwise when saving to storage, it will still be saved as `{"stepDescription": "step ..."}` based on the variable name, which will cause problems when reading as the `@JsonCreator` constructor is expecting `{"step ..."}`. So then we need to put in something like:
```java
@JsonValue
public String getStepDescription() {
    return stepDescription;
}
```

Reply:
alright, i'll update it